src/Acucar.java

```diff
@@ -0,0 +1,27 @@
+public class Acucar {
+    private String nivelAcucar;
+    private int nivelAcucarPadrao = 3;
+
```

Comment:
1. O que é esse número mágico 5? E por que a classe Acucar conhece ele?
2. o método selecionarNivelAcucar retorna o atributo nivelAcucar, mas não faz nada com ele. Ficou estranho

Reply:
E porque o nivelAcucar é uma String enquanto o padrão é um int? Eles não deveriam ter o mesmo tipo? (Seja qual for o tipo)